config/routes.rb

```diff
@@ -254,18 +254,52 @@
   end
 
   if SETTINGS[:unattended]
-    resources :config_templates, :except => [:show] do
-      member do
-        get 'clone'
-        get 'lock'
-        get 'unlock'
-      end
+    resources :provisioning_templates, :only => [] do
       collection do
-        get 'auto_complete_search'
         get 'build_pxe_default'
-        get 'revision'
       end
     end
+
+    constraints(:type => /.+/) do
```

Comment:
I'd like both of these to work
`template_path(:type => 'Ptable')`
`ptables_template_path`